packages/neuron-wallet/src/mock.ts

```diff
@@ -8,14 +8,15 @@ export const transactions = Array.from({
 
 export const transactionCount = 500
 
-export interface MockWallet {
+export interface Wallet {
   name: string
   id: string
   balance: number
   address: string
   publicKey: Uint8Array
   msg: string
   password: string
+  mnemonic: string
```

Comment:
Will mnemonic be cached?

Reply:
I merged from channel/index.ts.It`s temp.Keystore have not been confirmed.